schema/tables/etc_services.yml

```diff
@@ -0,0 +1,10 @@
+name: etc_services
+examples: >-
+  Identify the TCP and UDP ports associated to standard services.
```

Comment:
What should I expect to see? What should I expect to not see?

Reply:
This is a standard unix file. It's just useful if you want to find out what the port is for "Telnet" or whatever.